src/main/java/uk/gov/hmcts/reform/civil/service/FeatureToggleService.java

```diff
@@ -115,6 +115,10 @@ public boolean isCarmEnabledForCase(CaseData caseData) {
                                                         epoch, false);
     }
 
+    public boolean isGaForLipsEnabled() {
+        return this.featureToggleApi.isFeatureEnabled("GaForLips");
```

Comment:
no need for "this" here
https://stackoverflow.com/questions/2411270/when-should-i-use-this-in-a-class